e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -13,6 +13,12 @@ param(
     [Parameter(Mandatory)]
     [string] $GroupCertificatePassword,
 
+    [Parameter(Mandatory)]
```

Comment:
Since this isn't publicly available, can we not make it mandatory yet? Also, it feels like a tall order for anyone to run our E2E tests to have to set up a CA. I'd rather it be optional and the tests that need it just fail if it isn't set.